Update Release Timeline and Release Management Page (#338)
catenax-eV/catenax-ev.github.io: src/pages/timelines.md

[![Timeline for Release 26.06](@site/static/img/timelines/cx-timeline-26.06.png)](@site/static/img/timelines/cx-timeline-26.06.png)
*Timeline for Release 26.06*

[![Timeline for Release 26.09](@site/static/img/timelines/cx-timeline-26.09.png)](@site/static/img/timelines/cx-timeline-26.09.png)
*Timeline for Release 26.09*

---

### Catena-X Proteus, incl. Eclipse Tractus-X Release 27.03

Following the major release CX-Neptune, CX-Proteus is directly connected to its parent release, as Proteus is one of Neptune’s moons.
This minor release therefore represents continuity and close evolution within the established ecosystem rather than a change of direction.
It focuses on strengthening and refining what was introduced with CX-Neptune, ensuring stability, coherence, and improved interoperability across the ecosystem.
In this sense, CX-Proteus acts as a natural extension of its major release, orbiting around its core innovations and gradually maturing them.

:::info

Starting with the **Catena-X 27.03 release**, the standards release process will follow a more flexible and continuous approach.

> As part of the introduction of this new release process with the goal of achieving **operational excellence**, the **26.12 release was skipped**. As a result, **Proteus consists of only one release part, 27.03**.

Reviewer: ```suggestion
As part of the introduction of this new release process with our commitment to further elevate **operational excellence**, the **26.12 release was skipped**. As a result, **Proteus consists of only one release part, 27.03**.
```
Otherwise it sounds like, we don't have operational excellence yet.

---

### Standard Review & TX Development

The Standard Review & Eclipse Tractus-X Development phase runs in parallel for Catena-X and Eclipse Tractus-X, with each initiative focusing on different aspects of the release process.

For Catena-X, this phase is dedicated to the review and validation of standard candidates. In parallel, the Eclipse Tractus-X community continues the development, implementation, and integration of the corresponding open-source components.
For details on the activities taking place within Eclipse Tractus-X during this phase, please refer to the [Eclipse Tractus-X](#eclipse-tractus-x) section.

---

#### Standard Application Window

The Standard Application Window is the overarching timeframe covering all formal review, quality assurance, and Opt-Out activities of the release cycle. Within this window, standards are validated, reviewed by experts, and prepared for their transition into official Catena-X standards.

---

#### Quality Assurance Phase

> The Quality Assurance Phase represents the **central review and validation stage** of the Catena-X release cycle. With the start of this phase, the standard owners formally hand over their completed standards to the **TC4S (Technical Committee for Standardization)**, which assumes coordination of the entire review process up to the final release decision.

Reviewer: ```suggestion
The Quality Assurance Phase represents the **central review and validation stage** of the Catena-X release cycle. With the start of this phase, the standard owners formally hand over their completed standards drafts to the **TC4S (Technical Committee for Standardization)**, which assumes coordination of the entire review process up to the final release decision.
```

> From this point on, all standard candidates enter a **structured review and validation process**. Feedback can still be incorporated, but only in **close alignment with TC4S and the assigned reviewing experts**.

Reviewer: ```suggestion
From this point on, all standard drafts enter a **structured review and validation process**. Feedback can still be incorporated, but only in **close alignment with TC4S and the assigned reviewing experts**.
```

To my understanding, at this point the documents are just `drafts`, only after approval of QG 2 with entering the O'pt Out they become `candidates`?

The objective of this phase is to ensure that every standard candidate is **complete, consistent, interoperable, and aligned with the quality expectations** of the Catena-X ecosystem. To achieve this, TC4S orchestrates a structured review involving the responsible committees and domain experts, each contributing their specific expertise to the evaluation.

> At the end of the phase, the **overarching committee formally confirms** that the standard fulfills the expectations defined by the responsible Expert Group and is ready to proceed to the next stage of the release process.

Reviewer: ```suggestion
At the end of the phase, the **overarching committee formally confirms** that the standard drafts fulfills the expectations defined by the responsible Expert Group and is ready to proceed to the next stage of the release process as standard candidates.
```

:::info

> With the start of the Quality Assurance Phase, all review feedback becomes **normative**. Mandatory findings must be resolved before a standard can proceed. Standard candidates with unresolved mandatory findings are considered **not release-ready** and will be excluded from the release.

Reviewer: ```suggestion
With the start of the Quality Assurance Phase, all findings must be resolved before a standard can proceed. Standard drafts with unresolved mandatory findings can not become standard candidates and are considered as **not release-ready**, which will be excluded from the release.
```

:::  

---

##### Quality Gate 1

> **Quality Gate 1** marks the first formal checkpoint within the Quality Assurance Phase. Its purpose is to ensure that only standard candidates aligned with the **values, goals, and quality principles** of the Catena-X data ecosystem proceed further in the release process. Each standard is systematically evaluated against defined criteria, focusing on **completeness, relevance, redundancy, and conformance**.

Reviewer: ```suggestion
**Quality Gate 1** marks the first formal checkpoint within the Quality Assurance Phase. Its purpose is to ensure that only standard drafts aligned with the **values, goals, and quality requirements** of the Catena-X data ecosystem proceed further in the release process. Each standard is systematically evaluated against defined criteria, focusing on **completeness, relevance, redundancy, and conformance**.
```
